_analyzers/token-filters/common_gram.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: default
+title: common_grams
+parent: Token filters
+nav_order: 60
+---
+<!-- vale off -->
+# Common_grams token filter
+<!-- vale on -->
+The `common_grams` token filter in OpenSearch improves search relevance by keeping commonly occurring phrases (common grams) in the text. This is useful when dealing with languages or datasets where certain word combinations frequently occur and can impact the search relevance if treated as separate tokens.
```

Comment:
I think it would be good to explicitly specify here that this token filter generates unigrams and bigrams , if only common words are present.